airflow/providers/fab/auth_manager/security_manager/override.py

```diff
@@ -1506,14 +1506,15 @@ def add_user(
             user.username = username
             user.email = email
             user.active = True
+            self.get_session.add(user)
```

Comment:
This fixes the SQA 2.0 warning however I think we might could / should leave it as it is. SQA 2.0 complains about the User object that is being merged into a Session along the backref cascade. In SQA 2.0 this won't happen. But we add the User object explicit to session so IMO we should be fine without the change.

Reply:
I guess that is Models from the [FAB](https://github.com/dpgaspar/Flask-AppBuilder), which do not supports SA20 neither.

cc: @vincbeck maybe you have information about it?

Reply:
As we discussed [here](https://github.com/apache/airflow/pull/39299#issuecomment-2092597149) we try to fix all warnings. So IMO this change looks good.

Reply:
looks good.

Reply:
Oops! I missed this one, but yeah, looks good :)